### ISSUE TITLE
2 problems:  can not Auto type text-selection, and can not Select-all-text

### ISSUE THREAD

**Issue by dnknn** [ssborbis/ContextSearch-web-ext#281]
## 💥1 problem 👇
![209_08163342](https://user-images.githubusercontent.com/10995302/92452768-1205be00-f1f1-11ea-96b8-3f0eefb777cd.png)
￣￣￣￣￣￣￣￣￣￣￣
**Expected behavior:** 
	Changed by default. or Adding a ON/OFF option
☑ `autoPasteFromClipboard` , Still be able to `Select all text in the search bar when it receives focus`

---

## 💥2 problem👇
 
⚠ What is discussed here is :  the `CS` extension Popup of the browser toolbar

<details>
<summary>Instead of `Quick Menu` (yes, it can do)</summary>

![333](https://user-images.githubusercontent.com/10995302/92462207-c0fbc700-f1fc-11ea-8665-4a143ce2dd3d.gif)

</details>

　　　　So can you also add an option⚙ to `CS-Popup`?

![209_08180446](https://user-images.githubusercontent.com/10995302/92463009-d1f90800-f1fd-11ea-98b4-01d7a7291ec1.png)

　　　　So can you also add an option⚙ for `toolbar-Popup`?

## ⬜/☑ Auto type text-selection on the Search Bar for `toolbar-Popup`

Just like Bookmarklet: 　　Auto type the  `window.getSelection()`

`javascript:kw=prompt("⚠Auto type text-selection", window.getSelection()); if(kw!=null){window.open('URL1' +kw); window.open('URL2' +kw);}`

![222](https://user-images.githubusercontent.com/10995302/92461487-ee944080-f1fb-11ea-9488-4827b39cf29b.gif)

**Comment by ssborbis** [comment]
1. Fixed locally
2. You can achieve the same result by using autoCopy + autoPasteFromClipboard but you would like another option that doesn't require autoCopy?

**Comment by ssborbis** [comment]
2. It probably should be the default action when text is selected, so that's been changed

**Comment by dnknn** [comment]
> 1. Fixed locally

　　　　Hi! What means? sorry for my English.

> 2. ....

now I understand, `<Sidebar>` 〓 `<Tray-Popup>` , the logic processing of `<Quick Menu>` is a bit different, Such as Search Bar's🔍 and 🕗history

----
In addition, `autoCopy` seems to be invalid, 
But regarding the function of autoCopy, I have been using [Copy Better](https://chrome.google.com/webstore/detail/hpihdokfdmmghaclaojfpmbckkhjgebc/?hl=en-US) for many years, because it with notification. other [Auto Copy](https://chrome.google.com/webstore/detail/bijpdibkloghppkbmhcklkogpjaenfkg)

<details>
<summary>Some fork of Chromium browsers provide this feature themselves.
</summary>

![209_10212134](https://user-images.githubusercontent.com/10995302/92735014-9f8d0d80-f3ab-11ea-9198-932664008614.png)


</details>

----

In summary, I think only the first problem needs to be fixed.
![209_08163342](https://user-images.githubusercontent.com/10995302/92452768-1205be00-f1f1-11ea-96b8-3f0eefb777cd.png)
☑ `autoPasteFromClipboard` , Still be able to `Select all text in the search bar when it receives focus`

**Comment by ssborbis** [comment]
>        Fixed locally

>　　　　Hi! What means? sorry for my English.

I fixed the code on my computer but haven't updated the repository code yet.

**Comment by dnknn** [comment]
I specially downloaded the 🆕 repo to test, it can work.
